x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ssl/cert/CertificateInfoTests.java

```diff
@@ -22,8 +22,11 @@
 
 public class CertificateInfoTests extends ESTestCase {
 
+    private static final String selfSignedCertPath = "/org/elasticsearch/xpack/security/transport/ssl/certs/simple/testnode.crt";
+    private static final String rootSignedCertPath = "/org/elasticsearch/xpack/security/transport/ssl/certs/simple/openldap.crt";
```

Comment:
I was gonna comment that this is weird and ask you what's up, but I see you just copied in from below looks like? If you know, what's up with this? Having hardcoded paths in a test like this is kinda weird.

Actually, if we have that explanation, can we put it in a comment?

Reply:
Not sure I follow ..

Using absolute paths here is the full path to the test location
```
/home/jakelandis/workspace/elasticsearch/main/x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ssl/cert/CertificateInfoTests.java
```
and here is the the full path to the resource
```
/home/jakelandis/workspace/elasticsearch/main/x-pack/plugin/core/src/test/resources/org/elasticsearch/xpack/security/transport/ssl/certs/simple/openldap.crt
```
I think these are just really deep paths to the resources (not sure why , there is a bit amount ES specific plumbing to loading these resouces)

Reply:
Ah yeah I gotcha, not sure what I was thinking here. Nevermind!